astropy/stats/jackknife.py

```diff
@@ -0,0 +1,194 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from scipy.special import erfinv
```

Comment:
Now that it is out of the function, the import needs to be wrapped such as:
try:
    from opdep import Superclass
except ImportError:
    warn(AstropyWarning('opdep is not present, so <functionality below> will not work.'))
    class SuperClass(object): pass